src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -42,35 +42,55 @@
 	public static final String UNKNOWN_VIDEO_TYPEMIME = "video/mpeg";
 	public static final String UNKNOWN_IMAGE_TYPEMIME = "image/jpeg";
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
+	public static final String AUDIO_TRANSCODE = "audio/transcode";
+	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AC3_TYPEMIME = "audio/vnd.dolby.dd-raw";
+	public static final String AUDIO_ADPCM_TYPEMIME = "audio/x-adpcm";
 	public static final String AUDIO_ADTS_TYPEMIME = "audio/vnd.dlna.adts";
-	public static final String AUDIO_MP3_TYPEMIME = "audio/mpeg";
-	public static final String AUDIO_MP4_TYPEMIME = "audio/x-m4a";
-	public static final String AUDIO_WAV_TYPEMIME = "audio/x-wav";
-	public static final String AUDIO_WMA_TYPEMIME = "audio/x-ms-wma";
+	public static final String AUDIO_AIFF_TYPEMIME = "audio/aiff";
+	public static final String AUDIO_APE_TYPEMIME = "audio/x-ape";
+	public static final String AUDIO_ATRAC_TYPEMIME = "audio/x-sony-oma";
+	public static final String AUDIO_AU_TYPEMIME = "audio/basic";
```

Comment:
Just as a side note for the people checking this PR in some years...Nothing to change ;-)
As i don't have a good memory i stick it there an other possibility:

audio/x-au